File: movies/python/movies_client.py
import grpc
from movies_pb2 import AllMoviesRequest
from movies_pb2_grpc import AllMoviesStub

channel = grpc.insecure_channel("localhost:50051")
client = AllMoviesStub(channel)

request = AllMoviesRequest()

all_movies_response = client.get_all_movies(request)

print(all_movies_response)

File: movies/javascript/movies_client.js
const grpc = require('@grpc/grpc-js');
const protoLoader = require('@grpc/proto-loader');

const packageDefinition = protoLoader.loadSync('../../protobufs/movies.proto', {});
const moviesPackage = grpc.loadPackageDefinition(packageDefinition);


const client = new moviesPackage.AllMovies(
  // gRPC server to connect to
  'localhost:50051',

  // Use insecure mode
  grpc.credentials.createInsecure()
);

client.get_all_movies({}, (err, response) => {
  if (err) {
    console.log(err);
  } else {
    console.log(`From server`, JSON.stringify(response));
  }
});
